podcast-player/src/components/ui/appPagination.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Container, Pagination } from "@mui/material";
+
+const AppPagination = ({ setPage, pageCount }) => {
+  const handleChange = (page) => {
+    setPage(page);
+  };
+  return (
+    <Container sx={{ p: 0 }}>
+      <Pagination
+        onChange={(e) => handleChange(e.target.textContent)}
```

Comment:
This might be the cause of this pagination bug:

- visit a podcast page and observe the pagination buttons
- click next
- observe the episode list fails to render